pyseir/inference/model_fitter.py

```diff
@@ -210,9 +204,23 @@ def set_inference_parameters(self):
         """
         self.fit_params = self.DEFAULT_FIT_PARAMS
         # Update any state specific params.
-        for k, v in self.PARAM_SETS.items():
-            if self.state_obj.abbr in k:
-                self.fit_params.update(v)
+        initial_params_df = pd.read_csv("./pyseir/inference/world_according_to_pyseir.csv")
+
+        INITIAL_PARAM_SETS = [
+            "R0",
+            "t0",
+            "eps",
+            "t_break",
+            "eps2",
+            "t_delta_phases",
+            "test_fraction",
+            "hosp_fraction",
+            "log10_I_initial",
+        ]
+        if int(self.fips) in initial_params_df["fips"]:
+            this_fips_df = initial_params_df.loc[initial_params_df["fips"] == int(self.fips)]
+            for param in INITIAL_PARAM_SETS:
```

Comment:
This is fine, but why did you change it from the original? `self.fit_params.update(v)` is pretty clear, especially if you make the key the str fips index.

Reply:
No huge reason, and changing now would require refactoring and shouldn't be blocking. Feel free to change this later if you want